style.css

```
@@ -0,0 +1,58 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 10px;
+}
+
+header {
+    background: gray;
+    height: 100px;
+}
+
+main {
+    display: flex;
+    height: 990px;
+
+}
+
+main .main-nav {
+    float: left;
+    background: green;
+    flex: 1;
+}
+
+main .container {
+    flex: 7;
+    background: red;
+}
+
+.container .one {
+    background: yellow;
+    height: 350px;
+    margin: 20px;
+}
+
+.container .two {
+    background: pink;
+    height: 220px;
+    margin: 20px;
+}
+
+.container .three {
+    background: purple;
+    height: 300px;
+    margin: 20px;
+    border-style: solid;
+}
+
+main .main-nav .main-nav-contact {
+    background: orange;
+    position: absolute;
+    width: 180px;
+    height: 150px;
+    left: 25px;
+    top: 920px;
+}
```

You can add 
```suggestion
.main-nav-content {
    flex: 1
}
```

to correctly separate `.main-nav-content` from `.main-nav-contact`

---

Also, try to group css, e.g. everything regarding `.main-nav` should be in the same place